src/main/resources/META-INF/mods.toml

```diff
@@ -1,6 +1,6 @@
 
 modLoader="javafml" #mandatory
-loaderVersion="[40,)"
+loaderVersion="[41,)"
 license="MIT License"
 issueTrackerURL="https://github.com/eerussianguy/BlazeMap/issues" #optional
```

Comment:
I missed this in 1.18.2 as well

Reply:
Ah yep, that's not aligned with the right Forge version and also doesn't have a max value. Good catch!

Reply:
That's not it.
The issue tracker URL is wrong 😅